cg/meta/workflow/microsalt.py

```diff
@@ -60,16 +60,16 @@ def get_config_path(self, filename: str) -> Path:
 
     def get_trailblazer_config_path(self, case_id: str) -> Path:
         case_obj: models.Family = self.status_db.family(case_id)
-        order_id = case_obj.name
+        project_id: str = self.get_project(case_obj.links[0].sample.internal_id)
```

Comment:
```suggestion
        sample_obj: model.Sample = case_obj.links[0].sample
        project_id: str = self.get_project(sample_obj.internal_id)
```
hehe I have a little hard time back tracking in the sql relations. Added a suggestion to expose more what is going on